snippets/Google.Datastore.V1Beta3.Snippets/DatastoreClientSnippets.cs

```diff
@@ -187,5 +267,458 @@ public void Overview()
             var insertedKey = commitResponse.MutationResults[0].Key;
             // End snippet
         }
+
+        // Snippets ported from https://cloud.google.com/datastore/docs/concepts/entities
+
+        [Fact]
+        public void CreateEntity()
+        {
+            string projectId = _fixture.ProjectId;
+            string namespaceId = _fixture.NamespaceId;
+
+            // Snippet: CreateEntity
+            KeyFactory keyFactory = new KeyFactory(projectId, namespaceId, "Task");
+            Entity entity = new Entity
+            {
+                Key = keyFactory.CreateInsertionKey(),
+                ["type"] = "Personal",
+                ["done"] = false,
+                ["priority"] = 4,
+                ["description"] = "Learn Cloud Datastore",
+                ["percent_complete"] = 75.0
+            };
+            // End snippet
+        }
+
+        [Fact]
+        public void InsertEntity()
+        {
+            string projectId = _fixture.ProjectId;
+            string namespaceId = _fixture.NamespaceId;
+
+            // Snippet: InsertEntity
+            KeyFactory keyFactory = new KeyFactory(projectId, namespaceId, "Task");
+            Entity entity = new Entity
+            {
+                Key = keyFactory.CreateInsertionKey(),
+                ["type"] = "Personal",
+                ["done"] = false,
+                ["priority"] = 4,
+                ["description"] = "Learn Cloud Datastore",
+                ["percent_complete"] = 75.0
+            };
+            DatastoreClient client = DatastoreClient.Create();
+            CommitResponse response = client.Commit(projectId, Mode.NonTransactional, new[] { entity.ToInsert() });
+            Key insertedKey = response.MutationResults[0].Key;
+            // End snippet
+        }
+
+        [Fact]
+        public void LookupEntity()
+        {
+            string projectId = _fixture.ProjectId;
+            string namespaceId = _fixture.NamespaceId;
+            Key key = _fixture.LearnDatastoreKey;
+
+            // Snippet: LookupEntity
+            DatastoreClient client = DatastoreClient.Create();
+            LookupResponse response = client.Lookup(
+                projectId,
+                new ReadOptions { ReadConsistency = ReadConsistency.Eventual },
+                new[] { key });
+
+            Entity entity = response.Found[0].Entity;
+            // End snippet
+        }
+
+        [Fact]
+        public void UpdateEntity()
+        {
+            string projectId = _fixture.ProjectId;
+            string namespaceId = _fixture.NamespaceId;
```

Comment:
Correct - thanks for spotting. Will see if turning warnings up to max let's me find these in an easy way...